docs/classes/Path.rst

```diff
@@ -203,10 +203,10 @@ A `Path` object represents vector graphics as drawn by a pen. A path can be eith
             }
         }
 
-        var doc = mupdfjs.PDFDocument.openDocument(fs.readFileSync("test.pdf"), "application/pdf")
+        var doc = mupdf.PDFDocument.openDocument(fs.readFileSync("test.pdf"), "application/pdf")
```

Comment:
I'm slightly confused here, in the other file you included PDFDocument from mupdfjs?

Reply:
Yes, this should be mupdf.Document.openDocument as it's not using any of the mupdfjs extensions.